### PR TITLE
chore: Upgrade Sentry in v1, v2 & web

### PR DESCRIPTION
**Pull request by emrysal**
## What does this PR do?

Upgrade Sentry to v8

### CODE REVIEW

File: apps/api/v2/src/app.ts [hunk 1]
```diff
@@ -1,11 +1,9 @@
-import { getEnv } from "@/env";
 import { HttpExceptionFilter } from "@/filters/http-exception.filter";
 import { PrismaExceptionFilter } from "@/filters/prisma-exception.filter";
-import { SentryFilter } from "@/filters/sentry-exception.filter";
```
Comment (emrysal) [inline apps/api/v2/src/app.ts:4]: @ThyMinimalDev Was there any specific reason this was added that setupNestErrorHandler doesn't handle?

File: apps/api/v2/package.json [hunk 2]
```diff
@@ -39,8 +39,7 @@
     "@nestjs/platform-express": "^10.0.0",
     "@nestjs/swagger": "^7.3.0",
     "@nestjs/throttler": "^5.1.2",
-    "@sentry/node": "^7.86.0",
-    "@sentry/tracing": "^7.86.0",
```
Comment (emrysal) [inline apps/api/v2/package.json:43]: Deprecated

File: apps/api/v2/.prettierrc.js [hunk 3]
```diff
@@ -2,5 +2,6 @@ const rootConfig = require("../../../packages/config/prettier-preset");
 
 module.exports = {
   ...rootConfig,
+  importOrder: ["^./instrument", ...rootConfig.importOrder],
```
Comment (emrysal) [inline apps/api/v2/.prettierrc.js:5]: According to the Sentry Nest docs @ https://docs.sentry.io/platforms/javascript/guides/nestjs/#use - the import of instrument.ts should be the first thing that happens in the app.ts file; before any other imports. This rule override ensures that import "./instrument"; indeed happens first thing.

File: apps/api/v2/src/app.ts (hunk 1 above)
Comment (ThyMinimalDev) [reply apps/api/v2/src/app.ts:4]: no I think it was more of a fail on our side for not knowing setupNestErrorHandler exists, good one

**Review by zomars — APPROVED** [review]
LGTM

File: packages/core/sentryWrapper.ts [hunk 4]
```diff
@@ -11,55 +10,22 @@ For smaller loops, the cost incurred may not be very significant on an absolute
 considering that a million monitored iterations only took roughly 8 seconds when monitored.
 */
 
-const setUpMonitoring = (name: string) => {
```
Comment (keithwillcode) [inline packages/core/sentryWrapper.ts:14]: Why are we getting rid of this @emrysal?

**Pull request merged**